sdk/core/Azure.Core/tests/TransportFunctionalTests.cs

```diff
@@ -1015,6 +1015,52 @@ public async Task ServerCertificateCustomValidationCallbackIsHonored([Values(tru
             }
         }
 
+        [Test]
+        public async Task No100ContinueSentByDefault()
+        {
+            using (TestServer testServer = new TestServer(
+                       async context =>
+                       {
+                           Console.WriteLine(context.Request.Headers["Expect"]);
+                           Assert.True(context.Request.Headers["Expect"] == "");
+                           await context.Response.WriteAsync("");
+                       }))
+            {
+                var transport = GetTransport();
+                Request request = transport.CreateRequest();
+                request.Method = RequestMethod.Post;
+                request.Uri.Reset(testServer.Address);
+                request.Content = RequestContent.Create("Hello");
+                Response response = await ExecuteRequest(request, transport);
+
+                Assert.AreEqual(200, response.Status);
+            }
+        }
+
+        [Test]
+        public async Task CanSendExpect100Continue()
```

Comment:
Should we have a test where we add a different value from 100-continue?

Reply:
We do https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/tests/TransportFunctionalTests.cs#L250